components/search_results_item/index.test.ts

```diff
@@ -109,7 +132,7 @@ describe('components/SearchResultsItem/WithStore', () => {
                 editingPost: {},
             },
         },
-    };
+    } as unknown as GlobalState;
```

Comment:
I had to use an `unknown` here, not sure it's correct, but I didn't find another way.

Reply:
I believe this is correct; the type isn't fully compatible because its missing stuff at multiple levels, so it needs to be forced. You could do `} as DeepPartial<GlobalState>` or `const defaultState: DeepPartial<GlobalState> = {` instead, but then you'd just be playing whack-a-mole with all the other type errors.